firestore.rules

```diff
@@ -1,11 +1,11 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-
-    
-    match /{document=**} {
-      allow read: if false;
-      allow write: if true;
+    match /participant_responses/{study_id}/participants/{participant_id}/data/{document=**} {
+          allow read: if false
+          allow create, update: 
+          if participant_id in get(/databases/$(database)/documents/registered_studies/$(study_id)).data.registered_participants &&
+              exists(/databases/$(database)/documents/registered_studies/$(study_id));
```

Comment:
is this second one needed? if the first `get` works, mustn't the `second`?

Reply:
Yes, it wasn't needed and I deleted the line.